Flags/SL.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Sierra Leone | Flags</title>
+    <style>
+      #sl{

```

Comment:
Please change the id to full name of country. :) Make the same change in file name and image name.